From a66c666475450e2523e5c895e2538d0e6f3a5bce Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Tue, 17 Aug 2021 14:41:58 +0300
Subject: [PATCH 03/19] s32gen1: thermal: Fix static code issues

commit a66c666475450e2523e5c895e2538d0e6f3a5bce from
https://source.codeaurora.org/external/autobsps32/linux

Add 'static' keyword to members used only in this file.

Upstream-Status: Pending

Issue: ALB-7529
Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/thermal/s32_thermal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/s32_thermal.c b/drivers/thermal/s32_thermal.c
index 9eb83af432f2..1527400e98b6 100644
--- a/drivers/thermal/s32_thermal.c
+++ b/drivers/thermal/s32_thermal.c
@@ -101,7 +101,7 @@ struct fsl_tmu_chip {
 	int		 sites;
 };
 
-struct fsl_tmu_chip gen1_tmu = {
+static struct fsl_tmu_chip gen1_tmu = {
 	.has_enable_bits = false,
 	.has_clk = false,
 	.has_sites = true, /* This is how you tell if Gen1 or V2 */
@@ -110,7 +110,7 @@ struct fsl_tmu_chip gen1_tmu = {
 	.sites = 3,
 };
 
-struct fsl_tmu_chip v234_tmu = {
+static struct fsl_tmu_chip v234_tmu = {
 	.has_enable_bits = true,
 	.has_clk = true,
 	.has_sites = false,
-- 
2.17.1

